External/FEXCore/Source/Interface/Core/Dispatcher/Dispatcher.cpp

```diff
@@ -88,7 +88,247 @@ ArchHelpers::Context::ContextBackup* Dispatcher::StoreThreadState(FEXCore::Core:
   return Context;
 }
 
-void Dispatcher::RestoreThreadState(FEXCore::Core::InternalThreadState *Thread, void *ucontext) {
+void Dispatcher::RestoreFrame_x64(ArchHelpers::Context::ContextBackup* Context, FEXCore::Core::CpuStateFrame *Frame, void *ucontext) {
```

Comment:
I'm guessing this didn't change, but for reference putting code moves like this one into a dedicated patch helps confirm there are no slip-ups (git-diff can highlight moved lines in a different color, but only for patches that don't also modify close code).

Reply:
Split out to #2355 for the 64-bit save/restore.